### PR TITLE
update to include flatpak support

### PR DESCRIPTION
**Pull request by ellietheoracle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,12 @@ You can always install using the downloads (deb, rpm, tar) on the [releases page
 VSCodium is available in [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository) as package [vscodium-bin](https://aur.archlinux.org/packages/vscodium-bin/), maintained by [@plague-doctor](https://github.com/plague-doctor).
 
 #### <a id="flatpak"></a>Flatpak Option (Linux)
-VSCodium is not available as a Flatpak app, but [@amtlib-dot-dll](https://github.com/amtlib-dot-dll) has done significant work to package up the open source build of Visual Studio Code without telemetry, very similarly to VSCodium. That package is available [here](https://flathub.org/apps/details/com.visualstudio.code.oss) and the build repo is [here](https://github.com/flathub/com.visualstudio.code.oss).
+VSCodium is available as a Flatpak app, thanks to Peter Squicciarini who has packaged VSCodium. That package is available [here](https://flathub.org/apps/details/com.vscodium.codium) and the build repo is [here](https://github.com/flathub/com.vscodium.codium). if your distrobution has support for [flatpak](https://flathub.org), and you have enabled the [flathub repo](https://flatpak.org/setup/):
```
Comment (sonnyp) [inline README.md:75]: typo `distrobution` -> `distribution`

Comment (ellietheoracle) [reply README.md:75]: oops .

**Comment by akvadrako** [comment]
Thanks! The current README.md is quite confusing, since there is no indication in the `com.vscodium.codium` on flathub that it's published by a third party.

**Comment by ellietheoracle** [comment]
> Thanks! The current README.md is quite confusing, since there is no indication in the `com.vscodium.codium` on flathub that it's published by a third party.

i did fix this, in 6367d32

**Comment by stripedpajamas** [comment]
Thanks!!

**Pull request merged**